compiler/noirc_frontend/src/ast/expression.rs

```diff
@@ -522,6 +538,7 @@ impl Display for ExpressionKind {
             Quote(block) => write!(f, "quote {block}"),
             Comptime(block) => write!(f, "comptime {block}"),
             Error => write!(f, "Error"),
+            Resolved(_) => write!(f, "?Resolved"),
```

Comment:
Can this be seen by users?

Reply:
It shouldn't be, the Display impl on the AST is just for debugging in the compiler. Even then, we shouldn't ever be printing an AST with this temporary resolved node in it.